docs/towers.md

```diff
@@ -0,0 +1,73 @@
+# The Towers
+
+Towers are the multi-block structures that attack the enemies either by damaging them or affecting their stats.
+A tower is made up of at least three blocks:
+* a core block
+* an effector block
+* a targeter block
+
+## Cores
+
+The Cores of a tower define its power level.
+A tower needs power to run effectors and targeters that will allow the tower to target gooeys and affect these targets.
+Upgrades to a tower core increase the amount of power it can provide. 
+
+## Effectors
+
+Effectors define how a tower will affect targets.
+They drain the tower's power to realize the tower's effect on targeted enemies.
+
+Currently, there are the following effectors:
+
+| Effector       | Effect       |  Notes                                |
+-------------------------------------------------------------------------
+| DamageEffector | Plain Damage | direct damage only                    |
+| FireEffector   | Burn Damage  | damage over time, chance of spreading |
+| IceEffector    | Slow         |                                       |
+| PoisonEffector | Poison       | damage both direct and over time      |
+| StunEffector   | Stun         |                                       |
+| VisualEffector | Enlarge      | increases enemy visibility            | 
```

Comment:
this table does not render correctly. same for the one below: 

![image](https://user-images.githubusercontent.com/1448874/115159364-13566d80-a093-11eb-8c98-961989326e16.png)

Reply:
the pipe `|` is missing in the horizontal divider line.